api_app/tests/test_serializers.py

```diff
@@ -87,3 +90,126 @@ def test_user_serialization_missing_data(self):
         self.assertIsNone(serialized_data.validated_data.get('first_name'))
         self.assertIsNone(serialized_data.validated_data.get('last_name'))
         self.assertIsNone(serialized_data.validated_data.get('email'))
+
+
+class ProductSerializerTests(TestCase):
+
+    def setUp(self):
+        self.fake = Faker()
+        self.product_data = ProductFactory()
+        self.serializer_input_data = {
```

Comment:
setUpTestData() is called once at the beginning of the test run for class-level setup. You'd use this to create objects that aren't going to be modified or changed in any of the test methods.

setUp() is called before every test function to set up any objects that may be modified by the test (every test function will get a "fresh" version of these objects).